src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesFieldMethodAccess.java

```diff
@@ -0,0 +1,48 @@
+/*
+UnnecessaryParentheses
+tokens = EXPR, IDENT, NUM_DOUBLE, NUM_FLOAT, NUM_INT, NUM_LONG, \
+         STRING_LITERAL, LITERAL_NULL, LITERAL_FALSE, LITERAL_TRUE, ASSIGN, \
+         BAND_ASSIGN, BOR_ASSIGN, BSR_ASSIGN, BXOR_ASSIGN, DIV_ASSIGN, \
+         MINUS_ASSIGN, MOD_ASSIGN, PLUS_ASSIGN, SL_ASSIGN, SR_ASSIGN, STAR_ASSIGN, \
+         LAMBDA, TEXT_BLOCK_LITERAL_BEGIN, LAND, LITERAL_INSTANCEOF, GT, LT, GE, \
+         LE, EQUAL, NOT_EQUAL, UNARY_MINUS, UNARY_PLUS, INC, DEC, LNOT, BNOT, \
+         POST_INC, POST_DEC, INDEX_OP, DOT, LITERAL_NEW, LOR
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessaryparentheses;
+
+public class InputUnnecessaryParenthesesFieldMethodAccess {
+
+    private static int foo;
+    private static Main INSTANCE;
+
+    private static class Main {
+        private String name;
+
+        private boolean check() {
+            return false;
+        }
+    }
+
+    public static void main(String[] args) {
+        foo = (args[0]).length();
+        // violation above 'Unnecessary parentheses around expression.'
```

Comment:
please update all to be in this format(correct + incorrect ) :
```
foo = args[0].length();
foo = (args[0]).length();
// violation above 'Unnecessary parentheses around expression.'
```

it will help to spot valitidy of violation easier.